listenbrainz/db/dump_manager.py

```diff
@@ -280,6 +306,37 @@ def write_hashes(location):
             raise
 
 
+def sanity_check_dumps(location, expected_count):
+    """ Sanity check the generated dummps to ensure that none are empty
+
+    Args:
+        location (str): the path in which the dump archive files are present
+        expected_count (int): the number of files that are expected to be present
+    Return:
+        boolean: true if the dump passes the sanity check
+    """
+
+    count = 0
+    for file in os.listdir(location):
+        try:
+            dump_file = os.path.join(location, file)
+            print(dump_file)
+            if os.path.getsize(dump_file) == 0:
+                print("Dump file %s is empty!" % dump_file)
+                return False
+            count += 1
+        except IOError as e:
```

Comment:
`os.path.getsize()` may raise `OSError`.

In Python3, `os` module always raise `OSError` (but IOError is a deprecated alias), see https://www.python.org/dev/peps/pep-3151/#confusing-set-of-os-related-exceptions about this subject.

https://docs.python.org/3/library/os.path.html#os.path.getsize